deployment/helm/iot-edge-accelerator/templates/system-modules/data-gateway-module.yaml

```diff
@@ -18,9 +18,10 @@ spec:
         dapr.io/enabled: "true"
         dapr.io/app-id: "data-gateway-module"
         dapr.io/app-port: "80"
+        dapr.io/app-protocol: "grpc"
     spec:
       containers:
       - name: data-gateway-module
-        image: suneetnangia/distributed-az-iot-edge-datagatewaymodule:main-8db42cf6ce413a9676a24fb1f3aba083de1e6fe7
```

Comment:
Thanks for fixing it, I think I can make use of values.yaml instead to specify particular image/tag for release from main branch or dynamically update it in values.yaml during the release action on GH.